Project.toml

```diff
@@ -13,7 +13,7 @@ UnsafeArrays = "c4a57d5a-5b31-53a6-b365-19f8c011fbd6"
 [compat]
 Distances = "0.8.2, 0.9, 0.10"
 MLJModelInterface = "^0.3,^0.4, 1.0"
-StatsBase = "0.32, 0.33"
+StatsBase = "0.32, 0.33, 0.34"
 UnsafeArrays = "1"
 julia = "1.3"
```

Comment:
@ablaom  can you bump this to 1.6? and update the project toml with `Pkg.update()`?